packages/formspree-core/src/utils.ts

```diff
@@ -24,100 +22,14 @@ export const clientHeader = (givenLabel: string | undefined): string => {
   return `${givenLabel} ${label}`;
 };
 
-export const appendExtraData = (
+export function appendExtraData(
   formData: SubmissionData,
   prop: string,
   value: string
-) => {
+): void {
   if (formData instanceof FormData) {
     formData.append(prop, value);
   } else {
-    formData = Object.assign(formData, { [prop]: value });
-  }
-};
-
-type HandleSCAargs = {
-  stripePromise: Stripe;
-  response: Response;
-  responseData: any;
-  payload: {
-    paymentMethod: PaymentMethod;
-    error?: undefined;
-  };
-  data: SubmissionData;
-  fetchImpl: (
-    input: RequestInfo,
-    init?: RequestInit | undefined
-  ) => Promise<Response>;
-  request: {
-    method: string;
-    mode: 'cors';
-    body: string | FormData;
-    headers: {
-      [key: string]: string;
-    };
-  };
-  url: string;
-};
-
-export const handleSCA = async ({
-  stripePromise,
-  response,
-  responseData,
-  payload,
-  data,
-  fetchImpl,
-  request,
-  url,
-}: HandleSCAargs): Promise<SubmissionResponse> => {
-  const stripeResult = await stripePromise.handleCardAction(
-    responseData.stripe.paymentIntentClientSecret
-  );
-
-  // Handle Stripe error
-  if (stripeResult.error) {
-    return {
-      response,
-      body: {
-        errors: [
-          {
-            code: 'STRIPE_CLIENT_ERROR',
-            message: 'Stripe SCA error',
-            field: 'paymentMethod',
-          },
-        ],
-      },
-    };
-  } else {
-    if (!payload.paymentMethod.id) {
-      appendExtraData(data, 'paymentMethod', payload.paymentMethod.id);
-    }
-    appendExtraData(data, 'paymentIntent', stripeResult.paymentIntent.id);
-    appendExtraData(data, 'resubmitKey', responseData.resubmitKey);
```

Comment:
See https://github.com/formspree/formspree-js/pull/47/files#diff-e4b6d52183a02dd7b86db4d24b037520114a76c0dabb0ab8f5d842ab79998227L101-L104, these actually don't do anything since we just ignore it and `JSON.stringify` body from a different object literal.